armory/datasets/preprocessing.py

```diff
@@ -57,6 +58,28 @@ def carla_over_obj_det_dev(element, modality="rgb"):
     )
 
 
+@register
+def carla_video_tracking_dev(element, max_frames=None):
+    return carla_video_tracking(
+        element["video"], max_frames=max_frames, split="dev"
+    ), carla_video_tracking_labels(
+        element["video"],
+        (element["bboxes"], element["patch_metadata"]),
+        max_frames=max_frames,
+    )
+
+
+@register
+def carla_video_tracking_test(element, max_frames=None):
+    return carla_video_tracking(
+        element["video"], max_frames=max_frames, split="test"
+    ), carla_video_tracking_labels(
+        element["video"],
+        (element["bboxes"], element["patch_metadata"]),
+        max_frames=max_frames,
+    )
```

Comment:
I am assuming `max_frames` gets passed in as a `dataset_kwargs` but this is untested as I cannot find an example of it being used previously. Planning to circle back and test over the next couple days.

Reply:
it should be passed as a `preprocessor_kwarg`. See the configs in #1788 as an example

Reply:
Addressed in 9716c588a48d001b1afab6bbe06cb351c0fbe710. Tested working with `max_frames` 2 and 10. Throws error due to dimension mismatch when `max_frames=1`.